tox.ini

```diff
@@ -59,6 +59,6 @@ commands_pre =
     pip list
     pip check
 commands =
-    pytest --cov xsdba -m "not requires_atmosds" {posargs}
+    pytest --cov xsdba  {posargs}
```

Comment:
The --cov flag can be added to the `PYTEST_ADDOPTS` string above, actually. Cleaner.